proto/metadata.proto

```diff
@@ -36,4 +38,238 @@ message Metadata {
   }
   // Testbed on which the test is intended to run.
   Testbed testbed = 4;
+
+  message PlatformExceptions {
+    message Platform {
```

Comment:
proto messages don't have to be nested just because they're single use. It's sometimes recommended to not nest them unless they are conceptually bound to the parent. In this case, it sort of feels right to me to move Platform and Deviations out to the top-level, because they are concepts that can stand independently. But to keep PlatformExceptions nested, since it is sort of used as a kind of glorified map field. Make sense to you?

Reply:
done